micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/citizen/Home/UserProfile.js

```
@@ -573,71 +758,115 @@ const UserProfile = ({ stateCode, userType, cityDetails }) => {
-                <div>
-                  <a style={{ color: "orange", cursor: "default", marginBottom: "5", cursor: "pointer" }} onClick={TogleforPassword}>
-                    {t("CORE_COMMON_CHANGE_PASSWORD")}
-                  </a>
+                <div style={{ width: "100%" }}>
+                  {changepassword == false && !Digit.Utils.getOTPBasedLogin()? (
+                    <Button
+                      label={t("CORE_COMMON_CHANGE_PASSWORD")}
+                      variation={"teritiary"}
+                      onClick={TogleforPassword}
+                      style={{ paddingLeft: "0rem" }}
+                    ></Button>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)